src/ng2-flatpickr.component.ts

```diff
@@ -46,15 +47,15 @@ export class Ng2FlatpickrComponent implements AfterViewInit, ControlValueAccesso
 	@Input()
 	placeholder: string = "";
 
-  @Input()
+ 	@Input()
 	addClass: string = "";
 
 	@Input()
 	setDate: string | Date;
 
-  @Input()
-  get tabindex() { return this._tabindex; }
-  set tabindex( ti: number ) { this._tabindex = Number( ti ); }
+	@Input()
```

Comment:
Thank you for fixing the spacing here. It's been bothering me since I realized it was there.... may bad 😩